src/main/java/com/ylysenkova/movieland/web/util/MovieBuilder.java

```diff
@@ -0,0 +1,59 @@
+package com.ylysenkova.movieland.web.util;
+
+
+import com.ylysenkova.movieland.model.Country;
+import com.ylysenkova.movieland.model.Genre;
+import com.ylysenkova.movieland.model.Movie;
+import com.ylysenkova.movieland.web.dto.request.SaveMovieRequest;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MovieBuilder {
+
+    private SaveMovieRequest saveMovieRequest;
+    private Movie movie;
+    private List<Country> countryList = new ArrayList<>();
+    private List<Genre> genreList = new ArrayList<>();
+
+    public static MovieBuilder fromMovieRequest(SaveMovieRequest saveMovieRequest) {
+        MovieBuilder movieBuilder = new MovieBuilder();
+        movieBuilder.saveMovieRequest = saveMovieRequest;
+        return movieBuilder;
+    }
+
+
+    public MovieBuilder getMovie() {
+
+        movie = new Movie();
+        movie.setId(saveMovieRequest.getId());
+        movie.setNameRussian(saveMovieRequest.getNameRussian());
+        movie.setNameNative(saveMovieRequest.getNameNative());
+        movie.setYearOfRelease(saveMovieRequest.getYearOfRelease());
+        movie.setRating(saveMovieRequest.getRating());
+        movie.setPrice(saveMovieRequest.getPrice());
+        movie.setDescription(saveMovieRequest.getDescription());
+        if (movie.getGenres() == null) {
+            return this;
+        }
+        for (Integer genreId : this.saveMovieRequest.getGenres()) {
+            this.genreList.add(new Genre(genreId));
+        }
```

Comment:
I think if (movie.getGenres() == null)  is useless in this place, because you will return partially filled movie in case of empty list of genres.
If you want avoid null pointer exception then would be better wrap subsequent loop in if (movie.getGenres() != null)

Reply:
otherwise it returns NullPointerException :(

Reply:
Я ж тебе говорю, перемести в цикл в условие 
(movie.getGenres() != null)